docs/community-support.md

```diff
@@ -0,0 +1,70 @@
+# Community support
+
+For bugs, you can use the GitHub [issue tracker](https://github.com/prometheus-operator/kube-prometheus/issues/new/choose).
+
+For questions, you can use the GitHub [discussions forum](https://github.com/prometheus-operator/kube-prometheus/discussions).
+
+Many of the `kube-prometheus` project's contributors and users can also be found on the #prometheus-operator channel of the Kubernetes Slack.
+
+`kube-prometheus` is the aggregation of many projects that all have different
+channels to reach out for help and support. While this community strives at
+supporting all users, it is sometimes more efficient to reach out to the
+specific project instead.
+
+## Prometheus, Alertmanager, node_exporter
+
+For documentation, check the Prometheus [online docs](https://prometheus.io/docs/). There is a
+[section](https://prometheus.io/docs/introduction/media/) with links to blog
+posts, recorded talks and presentations. This [repository](https://github.com/roaldnefs/awesome-prometheus) 
+(not affiliated to the Prometheus project) has also a list of curated resources
+related to the Prometheus ecosystem.
+
+For questions, see the Prometheus [community page](https://prometheus.io/community/) for the various channels.
+
+There is also a #prometheus channel on the CNCF Slack.
+
+## prometheus-operator
```

Comment:
I would start with this first, as we still want folks to reach to us first as it might be a problem with the operator rather than Prometheus or alertmanager. So just move this to the first place maybe?

Reply:
:+1: